packages/-ember-data/tests/integration/store-test.js

```diff
@@ -990,53 +990,51 @@ module('integration/store - findAll', function(hooks) {
 module('integration/store - deleteRecord', function(hooks) {
   setupTest(hooks);
 
-  hooks.beforeEach(function() {
+  test('Using store#deleteRecord should mark the model for removal', function(assert) {
+    assert.expect(3);
+
     this.owner.register('model:person', Person);
-    this.owner.register('model:car', Car);
     this.owner.register('adapter:application', RESTAdapter.extend());
     this.owner.register('serializer:application', RESTSerializer.extend());
-  });
-
-  test('Using store#deleteRecord should mark the model for removal', function(assert) {
-    assert.expect(3);
 
     let store = this.owner.lookup('service:store');
-
     let person;
 
-    run(() => {
-      store.push({
-        data: {
-          type: 'person',
-          id: '1',
-          attributes: {
-            name: 'Tom Dale',
-          },
+    store.push({
+      data: {
+        type: 'person',
+        id: '1',
+        attributes: {
+          name: 'Tom Dale',
         },
-      });
-      person = store.peekRecord('person', 1);
+      },
     });
 
-    assert.ok(store.hasRecordForId('person', 1), 'expected the record to be in the store');
+    person = store.peekRecord('person', '1');
```

Comment:
Just a note when using push for a single-resource-document we can just use the returned record vs following with a peek. e.g.

```js
const person = store.push({ data: { ... } });
```